locales/en/plugin__kubevirt-plugin.json

```diff
@@ -543,12 +543,12 @@
   "Guest agent is required": "Guest agent is required",
   "Guest login credentials": "Guest login credentials",
   "Guest management": "Guest management",
-  "Guest system access log": "Guest system access log",
-  "Guest system access logs are disabled at cluster": "Guest system access logs are disabled at cluster",
-  "Guest system access logs are disabled at VirtualMachine": "Guest system access logs are disabled at VirtualMachine",
-  "Guest system access logs disabled at cluster": "Guest system access logs disabled at cluster",
-  "Guest system access logs not ready. a restart required": "Guest system access logs not ready. a restart required",
+  "Guest system log": "Guest system log",
   "Guest system log access": "Guest system log access",
+  "Guest system logs are disabled at cluster": "Guest system logs are disabled at cluster",
+  "Guest system logs are disabled at VirtualMachine": "Guest system logs are disabled at VirtualMachine",
+  "Guest system logs disabled at cluster": "Guest system logs disabled at cluster",
+  "Guest system logs not ready. a restart required": "Guest system logs not ready. a restart required",
```

Comment:
"a restart required" should be either "Restart required" or "A restart is required".

Reply:
@pcbailey's phrasing is an improvement for sure, but I think this message could be more specific.

Would either of these options make sense?

"To see guest system logs, restart the VM."
"Guest system logs are not available. Restart the VM."

If not, I would +1 changing "a restart required" to "Restart required."

Reply:
Changed to "Restart required".